### PR TITLE
Adding more default values to config_definitions

### PR DESCRIPTION
**Pull request by SourceDoctor**
add missing default properties to config_definitions.json

### CODE REVIEW

File: misc/config_definitions.json [hunk 1]
```diff
@@ -593,6 +593,10 @@
                 "mpls"
             ]
         },
+        "base_url": {
+            "type": "text",
+            "default": ""
```
Comment (murrant) [inline misc/config_definitions.json]: This one should not actually have a default as the default is set programatically.

**Comment by murrant** [comment]
Just one change.  Looks good, but just as a note this is not enough to add them to the webui, just allow them to be set via `lnms config:set`

File: misc/config_definitions.json (hunk 1 above)
Comment (SourceDoctor) [reply misc/config_definitions.json]: removed default value

**Comment by SourceDoctor** [comment]
@murrant 
defined position in WebUI now :-)

**Comment by SourceDoctor** [comment]
@murrant 
something else to do for integration?

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-58-release-changelog-november-2019/10224/1

**Pull request merged**